languages/alma-woocommerce-gateway-fr_FR.po

```diff
@@ -21,6 +21,14 @@ msgstr ""
 "X-Poedit-SearchPath-0: .\n"
 "X-Poedit-SearchPathExcluded-0: *.js\n"
 
+#: includes/class-alma-wc-generic-handler.php:181
+msgid ""
+"Alma \"Eligibility Widget\" (cart or product) already rendered on this page "
+"- Not displaying Alma"
+msgstr ""
+"\"Badge d'éligibilité\" Alma (produit ou panier) déjà affiché sur cette page "
+"- Alma ne s'affiche pas"
```

Comment:
> "Le badge Alma ne s'affiche pas" isn't better ?

Reply:
you're wright ... je referai une passe sur les contenus et traduction plus tard (quand les features les plus importantes seront en prod :) )